### PR TITLE
Add command with ability to run any job (ignoring due flag).

### PR DESCRIPTION
**Pull request by vanam**
Add command with ability to run any job (ignoring due flag).

Motivation: 
1) I don't need to fiddle with crontab in order to test job.
2) I can call any job outside of regular call times.

### CODE REVIEW

File: src/Command/ForceRunCommand.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ForceRunCommand extends Command
+{
+
+	/** @var string */
+	protected static $defaultName = 'scheduler:forceRun';
```
Comment (f3l1x) [inline src/Command/ForceRunCommand.php]: I prefer snake_case.

Comment (vanam) [reply src/Command/ForceRunCommand.php]: Fixed

Comment (f3l1x) [reply src/Command/ForceRunCommand.php]: Oh shit, I ment camel-case. Sorry. Too early for me. :-D

`scheduler:force-run`

**Comment by f3l1x** [comment]
Excellent. 👍

**Comment by petrparolek** [comment]
Please add information about this feature to readme

**Comment by f3l1x** [comment]
It's there https://github.com/contributte/scheduler/tree/master/.docs#console

**Comment by petrparolek** [comment]
sorry I overlooked it

**Pull request merged**